Scenarios.md

```diff
@@ -933,6 +983,16 @@
  * should upgrade the package
  * should upgrade where install location reports
 
+#### when upgrading a package with shims

```

Comment:
What about when upgrading a package that has existing shims with no shims in the upgrade? What's the expected behavior for that use case?

Reply:
The default behaviour in install and upgrade scenarios is to remove all shims for the package first, then install any shims if not prohibited by --noshims or --noshimsglobal.

So the existing shims will be removed and the new shims will not be installed.